### PR TITLE
Matches the latest table structure of obkv redis

### PR DESCRIPTION
**Pull request by lyxiong0**

### CODE REVIEW

File: command/server.go [hunk 1]
```diff
@@ -52,33 +52,33 @@ type DBInfo struct {
...
-			if err != nil {
-				log.Warn("command", ctx.TraceID, "fail to write string builder",
-					log.Errors(err), log.Int64("db", db))
-				return err
-			}
-		}
-	}
-	return nil
-}
+// func formatDBInfo(ctx *CmdContext, infoBuilder *strings.Builder) error {
```
Comment (WeiXinChan) [inline command/server.go]: maybe you should delete the unused code

File: command/utils.go [hunk 2]
```diff
@@ -216,42 +189,26 @@ func clientFlag2Str(flag conncontext.ClientFlag) string {
...
-			log.Warn("command", ctx.TraceID, "fail to get table info",
-				log.Errors(err), log.Int64("db", db), log.String("table name", tbName))
-			return nil, err
-		}
-		dbInfo.Keys += tbInfo.Keys
-		dbInfo.Expires += tbInfo.Expires
-	}
-	return dbInfo, nil
-}
+// func getDBInfo(ctx *CmdContext, db int64) (*DBInfo, error) {
```
Comment (WeiXinChan) [inline command/utils.go]: maybe you should delete the unused code

**Review by eemjwu — APPROVED** [review]
LGTM

**Comment by WeiXinChan** [comment]
LGTM

**Pull request merged**